### ISSUE TITLE
Open world with data

### ISSUE THREAD

**Issue by 3x1t-5tyl3** [Yellow-Dog-Man/Resonite-Issues#3189]
### Is your feature request related to a problem? Please describe.

Currently there is no easy, fast and reliable way of opening worlds with data.
Such as TRS or just arbitrary data really

### Describe the solution you'd like

A pflux node "open world with data" that'd allow me to send data associated with the user to that world

### Describe alternatives you've considered

Cloudvariables. Sadly those can be slow at times and rely on the internet and the cloud. Which might take so long to update that it makes no feasable sense to use them.

### Additional Context

This would be useful when you have game worlds or even parent/child sessions to teleport users to the right spot without an external server and it also being fast.

### Requesters

Xau, 3x1t_5tyl3

**Comment by Electronus** [comment]
Duplicate or at least highly related I think .https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1466

**Comment by 3x1t-5tyl3** [comment]
> Duplicate or at least highly related I think .[#1466](https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1466)

yeah this isnt tiyng it into a url, its just raw data and itentionally noy dynamic as it woud be lost post load in (atleast in my head lol)

**Comment by Electronus** [comment]
Yeah froox actually talks about doing it differently in the comment chain there. ![Image](https://github.com/user-attachments/assets/affd01d3-2c18-4094-b4a2-d4354b86d4fc)


This one is more generic and fits what he describes better, figure the other should either be reworked slightly or just merged into this one? not for me to decide. lol
